scoreboard/app/Http/Controllers/RoomController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Room;
+use Illuminate\Http\Request;
+
+class RoomController extends Controller
+{
+    public function show($room) {
+        $room = Room::where('name', $room)->firstOrFail();
```

Comment:
Why redefine `$room`? Is the argument a string? Better to name the argument something different, like `$candidate`. (in multiple places)